src/Shell/CleanOldNotifsShell.php

```diff
@@ -37,10 +37,26 @@ public function main()
     {
         $XTime = time() - 60 * 24 * 3600;
         $conditions = array('Notifications.created <' => date('Y-m-d H:i:s', $XTime));
-        if (!$this->Notifications->deleteAll($conditions)) {
+
+        if ($this->Notifications->find('all', array('conditions' => $conditions))->count() === 0) {
+            // Check if there are any notifications to delete
+            Log::write(
+                'warning',
```

Comment:
I've changed this to info - warning goes to stderr and that should indicate problem, but there is really none here...